### PR TITLE
NOISSUE - Move CLI documentation from getting started guide to separate page

### PR DESCRIPTION
**Pull request by blokovi**
Signed-off-by: Ivan Milošević <iva@blokovi.com>

### What does this do?

Move CLI documentation from getting started guide to separate page

### Which issue(s) does this PR fix/relate to?
No issue

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

File: docs/cli.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+#### Retrieve All Things
+
+```
+./mainflux-cli things get all --offset=1 --limit=5 <user_auth_token>
+```
+
+#### Retrieve Thing By ID
+
+```
+./mainflux-cli things get <thing_id> --offset=1 --limit=5 <user_auth_token>
```
Comment (anovakovic01) [inline docs/cli.md]: `offset` and `limit` don't make sense when thing ID is passed.

File: docs/cli.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+#### Retrieve All Channels
+
+```
+./mainflux-cli channels get all --offset=1 --limit=5 <user_auth_token>
+```
+
+#### Retrievie Channel By ID
+
+```
+./mainflux-cli channels get <channel_id> --offset=1 --limit=5 <user_auth_token>
```
Comment (anovakovic01) [inline docs/cli.md]: `offset` and `limit` don't make sense when channel ID is passed.

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Review by anovakovic01 — APPROVED** [review]

**Pull request merged**